earth_enterprise/rpms/opengee-fusion/post-install.sh

```diff
@@ -196,13 +191,14 @@ final_assetroot_configuration()
     else
         "$BASEINSTALLDIR_OPT/bin/geselectassetroot" --assetroot "$ASSET_ROOT"
 
-        mkdir -p "$BASEINSTALLDIR_OPT/share/tutorials"
         "$BASEINSTALLDIR_OPT/bin/geconfigureassetroot" --addvolume \
             "opt:$BASEINSTALLDIR_OPT/share/tutorials" --noprompt --nochown
         if [ $? -eq 255 ]; then
-            echo -e "The geconfigureassetroot utility has failed on attempting"
-            echo -e "to add the volume 'opt:$BASEINSTALLDIR_OPT/share/tutorials'."
-            echo -e "This is probably due to a volume named 'opt' already exists."
+            cat <<END
+The geconfigureassetroot utility has failed on attempting
+to add the volume 'opt:$BASEINSTALLDIR_OPT/share/tutorials'.
+This is probably due to a volume named "opt"'s already existing.
```

Comment:
Minor: I think it would be less awkward to say "This is probably because a volume named "opt" already exists."